ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/vr/RDXVRModeManager.java

```diff
@@ -165,21 +172,6 @@ public void update()
       if (kinematicsStreamingMode != null)
       {
          kinematicsStreamingMode.update(mode == RDXVRMode.WHOLE_BODY_IK_STREAMING);
-
-         // Fade robot graphics
-         if (kinematicsStreamingMode.isStreaming() && stereoVision.isEnabled())
-         { // Fade graphics out if in stereo vision mode while streaming
-            kinematicsStreamingMode.visualizeIKPreviewGraphic(false);
-            robotVisualizer.fadeVisuals(0.0f, 0.01f);
-         }
-         else if (mode == RDXVRMode.WHOLE_BODY_IK_STREAMING && (stereoVision.getDisabledNotification().poll()
-                                                                || kinematicsStreamingMode.getStreamingDisabledNotification().poll()
-                                                                || robotVisualizer.isFading()))
-         { // Fade graphics in if either stereo vision or streaming are exited
-            robotVisualizer.fadeVisuals(1.0f, 0.05f);
-            if (!robotVisualizer.isFading())
-               kinematicsStreamingMode.visualizeIKPreviewGraphic(true);
-         }
```

Comment:
We don't support stereo vision anymore and this logic was causing bugs and visual issues so I removed it.